nixos/release-combined.nix

```diff
@@ -63,24 +65,24 @@ in rec {
         nixos.tests.chromium.x86_64-linux or []
         (all nixos.tests.firefox)
         (all nixos.tests.firewall)
-        (all nixos.tests.gnome3)
+        (except ["aarch64-linux"] nixos.tests.gnome3)
         nixos.tests.installer.zfsroot.x86_64-linux or [] # ZFS is 64bit only
```

Comment:
Though this is [already documented here](https://github.com/NixOS/nixpkgs/blob/9265c964a8a970286e6a4c2adcafc29cc83c253a/nixos/tests/all-tests.nix#L83) why it fails.

```
# libsmbios is unsupported on aarch64
```

Reply:
I mean the aarch64 + zfs test :)

Reply:
```
~/tmp/nixpkgs/nixpkgs 130 $ time nix-instantiate nixos/release-combined.nix
warning: dumping very large path (> 256 MiB); this may run out of memory
error: Non-EFI boot methods are only supported on i686 / x86_64
(use '--show-trace' to show detailed location information)
Command exited with non-zero status 1
```

This would be part of the work needed.